verification-annotations/src/verifier_std.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 The Propverify authors
```

Comment:
If I understand this correctly, this crate exists only to allow us to let us print the meta-annotations to support "should_panic".
In particular, I think it only exists to avoid linking errors?

So could we change every function body to just "panic()"?